app/components/Search.js

```diff
@@ -1,75 +1,61 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import {
+  ActivityIndicator,
   Animated,
   Alert,
   Image,
   ScrollView,
   StyleSheet,
-  Switch,
   Text,
-  TextInput,
   TouchableHighlight,
   View
 } from 'react-native';
 
-
-
 import userContainer from '../containers/userContainer';
 import meteorContainer from '../containers/meteorContainer';
 import Profile from './Profile';
 import Visuals from './Visuals';
 import Row from './Row';
 
-class Search extends Component{
+class Search extends Component {
   constructor (props) {
-   super(props);
-   this.state = {
-     searchTerm: null,
-   };
- }
+    super(props);
+    this.state = {
+      searchTerm: null,
+      isLoading: false,
+    };
+  }
 
- render() {
-   const { user, meteors } = this.props;
-   if(user) {
-     return (
-       <Image source={require('../assets/space-bkgd.png')}
+  render() {
+    const { user, meteors } = this.props;
+    if(user) {
+      return (
+        <Image source={require('../assets/space-bkgd.png')}
           style={styles.container}>
-        <Text style={styles.text}>Search Page</Text>
-        <TouchableHighlight
-          style={styles.button}
-          onPress={() => this.props.navigator.push({
-            component: Profile,
-            title: "Profile"
-          })}
-        >
-          <Text style={styles.buttonText}>Profile</Text>
-        </TouchableHighlight>
-        <TouchableHighlight
-          style={styles.button}
-          onPress={() => this.props.navigator.push({
-            component: Visuals,
-            title: "Visuals"
-          })}
-        >
-          <Text style={styles.buttonText}>Go To Visualization</Text>
-        </TouchableHighlight>
-        <TouchableHighlight
-          style={styles.button}
-          onPress={this._onCallApi.bind(this)}
-        >
-          <Text style={styles.buttonText}>Search</Text>
-        </TouchableHighlight>
-        <ScrollView
-          style={styles.scrollView}>
-          {meteors.map(function(meteor, i) {
-            return <Row key={i} meteor={meteor} />}
-          )}
-        </ScrollView>
-      </Image>
-     )
-   }
-   return (null)
+          <TouchableHighlight
+            style={styles.button}
+            onPress={this._onCallApi.bind(this)}
+          >
+            <Text style={styles.buttonText}>Search for Meteors</Text>
+          </TouchableHighlight>
+          {/* {spinner = this.state.isLoading ?
+            (<ActivityIndicator
+                style={styles.spinner}
+                size='large'
+                color='red'
+            /> ) : */}
```

Comment:
This is to have a spinner to show data loading when API call made. Needs more logic to know when data returns from NASA.